### PR TITLE
Asn/test containers maven lifecycle

### PR DESCRIPTION
**Pull request by artnaseef**
DRAFT - for discussion

### CODE REVIEW

**Comment by j-white** [comment]
+1 from me

File: minion-gateway/docker-it/src/test/resources/org/opennms/horizon/minion-gateway.feature [hunk 1]
```diff
@@ -0,0 +1,56 @@
+Feature: Minion Gateway RPC Request Processing
+
+  Background: Common Test Setup
+    Given Internal GRPC Port in system property "application-internal-grpc-port"
```
Comment (splatch) [inline minion-gateway/docker-it/src/test/resources/org/opennms/horizon/minion-gateway.feature:4]: I think for consistency reasons both grpc port and host should be configurable. This way scenario can be adopted to external systems to verify behavior of deployed installations.

**Review by splatch — APPROVED** [review]
:+1: LGTM with just one small nitpick

File: minion-gateway/docker-it/src/test/resources/logback.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
```
Comment (splatch) [inline minion-gateway/docker-it/src/test/resources/logback.xml:2]: Looks like we have both log4j and slf4j at the same time - is it intentional?

**Pull request merged**